dev/gallery.tsx

```diff
@@ -179,23 +149,6 @@ export function Gallery() {
                     <Strut size={spacing.xSmall_8} />
                     <label htmlFor={searchId}>Search Types</label>
                 </View>
-                <View style={styles.headerItem}>
-                    <MultiSelect
-                        id={flagsId}
-                        onChange={setMafsFlags}
-                        selectedValues={mafsFlags}
-                    >
-                        {MafsGraphTypeFlags.map((flag) => (
-                            <OptionItem
-                                key={flag}
-                                value={flag}
-                                label={capitalize(flag)}
-                            />
-                        ))}
-                    </MultiSelect>
-                    <Strut size={spacing.xSmall_8} />
-                    <label htmlFor={flagsId}>Mafs Flags</label>
-                </View>
```

Comment:
There's no "turning on mafs flags" now, so I'm removing the dropdown for doing that.